Firmware/printer.cfg

```diff
@@ -34,22 +59,23 @@ uart_pin: PC11
 tx_pin: PC10
 uart_address: 0
 interpolate: True
-run_current: 0.5        # For V0.1 spec NEMA14 40Ncm
+run_current: 0.5
 hold_current: 0.25
 sense_resistor: 0.110
-stealthchop_threshold: 0
-
+stealthchop_threshold: 0                                            # Set to 9999 to turn off stealthchop
```

Comment:
@newphreak this comment isn't right:

>  "# Set to 9999 to turn off Stealthchop" 

Stealthchop is **OFF** by default - the default value for `stealthchop_threshold` is 0.

Specifying a non-zero value `stealthchop_threshold` turns Stealthchop **ON** for velocities **LOWER** than it.

Reply:
eek, that is totally correct sir.